packages/demo-plugin-bar/package.json

```
@@ -42,7 +42,9 @@
   "devDependencies": {
     "babel-core": "6.26.0",
     "babel-eslint": "7.2.3",
-    "babel-loader": "7.1.2",
+    "v": "a",
```

```suggestion
    "v": "a@1.2.3",
```

it should be different version